src/organisms/forms/index.jsx

```diff
@@ -1,148 +1,163 @@
-				name="Size"
-				control={control}
-				render={({ field }) => (
-					<TextField
-						id="outlined-basic"
-						label="Size of your space"
-						variant="outlined"
-						size="large"
-						margin="normal"
-						{...field}
-					/>
-				)}
-			/>
-			<Controller
-				name="Number"
-				control={control}
-				render={({ field }) => (
-					<TextField
-						id="outlined-basic"
-						label="Number of possible buddies"
-						variant="outlined"
-						size="large"
-						margin="normal"
-						{...field}
-					/>
-				)}
-			/>
-			<Controller
-				name="Location"
-				control={control}
-				render={({ field }) => (
-					<TextField
-						id="outlined-basic"
-						label="Location"
-						variant="outlined"
-						size="large"
-						margin="normal"
-						{...field}
-					/>
-				)}
-			/>
-			<Controller
-				name="Soil quality"
-				control={control}
-				render={({ field }) => (
-					<TextField
-						id="outlined-basic"
-						label="Soil quality"
-						variant="outlined"
-						size="large"
-						margin="normal"
-						{...field}
-					/>
-				)}
-			/>
-			<Controller
-				name="Contact Info"
-				control={control}
-				render={({ field }) => (
-					<TextField
-						id="outlined-basic"
-						label="Contact Info"
-						variant="outlined"
-						size="large"
-						margin="normal"
-						{...field}
-					/>
-				)}
-			/>
+		<>
+			<form onSubmit={handleSubmit(onSubmit)}>
+				<Controller
+					name="description"
+					control={control}
+					render={({ field }) => (
+						<TextField
+							label="Find a welcoming title"
+							variant="outlined"
+							size="large"
+							margin="normal"
+							{...field}
+						/>
+					)}
+				/>
+				<Controller
+					name="size"
+					control={control}
+					render={({ field }) => (
+						<TextField
+							label="Size of your space"
+							variant="outlined"
+							size="large"
+							margin="normal"
+							{...field}
+						/>
+					)}
+				/>
+				<Controller
+					name="number"
+					control={control}
+					render={({ field }) => (
+						<TextField
+							label="Number of possible buddies"
+							variant="outlined"
+							size="large"
+							margin="normal"
+							{...field}
+						/>
+					)}
+				/>
+				<Controller
+					name="location"
+					control={control}
+					render={({ field }) => (
+						<TextField
+							label="Location"
+							variant="outlined"
+							size="large"
+							margin="normal"
+							{...field}
+						/>
+					)}
+				/>
+				<Controller
+					name="soilQuality"
+					control={control}
+					render={({ field }) => (
+						<TextField
+							label="Soil quality"
+							variant="outlined"
+							size="large"
+							margin="normal"
+							{...field}
+						/>
+					)}
+				/>
+				<Controller
+					name="contactInfo"
+					control={control}
+					render={({ field }) => (
+						<TextField
+							label="Contact Info"
+							variant="outlined"
+							size="large"
+							margin="normal"
+							{...field}
+						/>
+					)}
+				/>
```

Comment:
this controllers are pretty repeating, you could try building a component that handles all of your needed cases